sale_order_type/tests/test_sale_order_type.py

```diff
@@ -50,6 +50,14 @@ def setUp(self):
         self.immediate_payment = self.env.ref("account.account_payment_term_immediate")
         self.sale_pricelist = self.env.ref("product.list0")
         self.free_carrier = self.env.ref("account.incoterm_FCA")
+        self.analytic_account = self.env["account.analytic.account"].create(
```

Comment:
You should put these changes in a separate commit with the message:

```
[IMP] sale_order_type: Don't depend on demo data for tests
```

for not mixing unrelated things in the diff.